server/src/main/java/io/crate/types/DoubleType.java

```diff
@@ -156,6 +156,8 @@ public Double implicitCast(Object value) throws IllegalArgumentException, ClassC
             return null;
         } else if (value instanceof Double d) {
             return d;
+        } else if (value instanceof Float f) {
```

Comment:
Does this fix deserve its own changelog message and commit?
or it's only effective in combination with the other changes?

Reply:
I think we don't need it

We have below Number handling which covers Float (and it also looks cheaper)
```
 else if (value instanceof Number n) {
            return n.doubleValue();
        }
```

Reply:
+1 thx @BaurzhanSakhariev

Reply:
Thank you I had this part as an experiment and forgot to remove it. This prevents `1.1` to be `1.100000023841858` from float to double conversion, https://github.com/crate/crate/pull/16623.